packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -267,6 +269,8 @@ class SampleApp extends Component {
   render() {
     return (
       <View style={styles.container}>
+
+        <MyComp/>
```

Comment:
Is this meant to be in this change?

Reply:
I'm good with this change going in.  It makes testing / demoing fast refresh easier.